src/enumerable.cr

```diff
@@ -220,6 +220,15 @@ module Enumerable(T)
     end
   end
 
+  # Returns a copy of `self` with all `nil` elements removed.
```

Comment:
This should say that it returns a new Array. A copy of self would imply it's the same type, but that is only true if the receiver is an Array.

Reply:
ouch! copy/paste error 🙈